purchase_work_acceptance/readme/USAGE.rst

```diff
@@ -0,0 +1,5 @@
+**Work Acceptance**
```

Comment:
Please add more detail about how to use this. We can also talk about invoice and picking.